app/lib/actions/candlepin/import_pool_handler.rb

```diff
@@ -35,14 +37,19 @@ def reindex_consumer(message_handler)
         subscription_facet = message_handler.subscription_facet
         sub_status = message_handler.sub_status
         uuid = message_handler.consumer_uuid
+
         if subscription_facet && sub_status
           @logger.debug "re-indexing content host #{subscription_facet.host.name}"
-          subscription_facet.update_subscription_status(sub_status)
-          subscription_facet.update_compliance_reasons(message_handler.consumer_reasons)
+          if message_handler.subject.match /purpose/
```

Comment:
I would spell out the whole name here, mostly just so people know its just trying to capture the one message